### DISCUSSION TITLE
Dark mode - Possible to change default background color from theme.colors.dark[7] to theme.colors.dark[9]?

### DISCUSSION THREAD

**Discussion by totaldekadens** [mantinedev/mantine#2651] [Q&A]
Hi! I'm very new and have a little problem with customizing the dark theme. Is it possible to change the default shades in dark mode and light mode? E.g from theme.colors.dark[7] to theme.colors.dark[9]? 

I tried to change the background color and color in body (tried * too) in "globalStyles" but it seems not to be able to override my header and footer components (They have not been set with any color manually earlier). But they still choose the default theme shades.  But my new settings do override the main Container with the new colors. So it seems to work at some of your components. 

Is the check if colorscheme is dark do this else do that the way to go or is there another way to go around this?

**Comment by rtivital** [comment]
You can set body background color with global styles – https://mantine.dev/styles/global-styles/
To change background color of components use sx prop or styles api – https://mantine.dev/styles/styles-api/
